src/components/WideRHPContextProvider/index.tsx

```diff
@@ -153,9 +153,12 @@ function WideRHPContextProvider({children}: React.PropsWithChildren) {
             return false;
         }
 
-        // Check the focused route to avoid glitching when quickly close and open RHP.
-        if (wideRHPRouteKeys.length > 0 && !wideRHPRouteKeys.includes(focusedRoute.key) && isRHPLastRootRoute && focusedRoute.name !== SCREENS.SEARCH.REPORT_RHP) {
-            return true;
+        // We want to show the second overlay only if
+        // 1. The currently focused route is not the wide report RHP
+        // 2. A wide report RHP exists on the navigation stack
```

Comment:
Can you change the comment to explain _why_ we want to show the second overlay in these cases please?

Reply:
Can I append this to the existing comment?
```
so that the wide RHP beneath the normal RHP is overlaid
```